package/com.unity.formats.usd/Runtime/Scripts/IO/Skel/SkeletonImporter.cs

```diff
@@ -106,6 +106,10 @@ public static class SkeletonImporter {
       }
     }
 
+    /// <summary>
+    /// Unity expect bind transforms to be the bone's inverse transformation matrix.
+    /// USD don't do that, so this function do it for us, and so prep the data to be used in Unity.
+    /// <summary>
```

Comment:
English bugs :)

/// Unity **expects** bind transforms to be the bone's inverse transformation matrix.
/// USD *doesn't* do that, so this function **does** it for us, **prepping** the data to be used in Unity.